src/main/java/org/openstreetmap/atlas/geography/atlas/change/serializer/FeatureChangeGeoJsonSerializer.java

```diff
@@ -207,15 +207,14 @@ else if (source instanceof Relation)
         return result;
     };
 
-    private static final String NULL = "null";
```

Comment:
Would this be used to add null as a String? As in

```json
{"name": "null"}
```

instead of,

```json
{"name": null}
```

Reply:
That is what it was used for before this PR. Now we no longer have need for it. Interestingly enough, it seems like gson is not even adding the null fields. So while the code explicitly sets things to null like `{"tags": null}`, they end up just being excluded from the final GeoJSON object.

Reply:
Thank you @lucaspcram so will you be removing this constant?

Reply:
Yes. It's removed by this PR!